extract download command (#8)
elastic/universal-profiling-documentation: README.md

1. Copy your deployment's Cloud ID from the Cloud console.
   ![cloud ID](./img/cloud-id.png)
1. On your Linux machine, open a terminal to execute the following steps.
1. Download and extract the `elastic-profiling` CLI by replacing the `<PROVIDED_URL>` placeholder in the following example with the URL you received in your private beta onboarding call:
   ```bash
   wget -O- <PROVIDED_URL> | tar xz

>    chmod +x

Reviewer: we still want the file name in this command,

```suggestion
   chmod +x elastic-profiling
```

Reviewer: The name of the file is going to be fixed and it's always going to be extracted in the local folder with the previous command, so that will work.

1. Copy your deployment's Cloud ID from the Cloud console.
   ![cloud ID](./img/cloud-id.png)
1. On your Linux machine, open a terminal to execute the following steps.

> 1. Download and extract the `elastic-profiling` CLI, which will be provided in the private beta onboarding call. 

Reviewer: Wording is a little weird.
What do you think of 

```suggestion
1. Download and extract the `elastic-profiling` CLI from the URL provided in the private beta onboarding call.
   ```bash
    wget -O- <PROVIDED_URL> | tar xz
    chmod +x
   ```
```

Because we still want to provide a straight command to run

Author: @mdbirnstiehl  - could you review this statement and make appropriate adjustments in the grammar, please?

> 1. Download and extract the `elastic-profiling` CLI, which will be provided in the private beta onboarding call. 
>   [Sign-up for the private beta if you haven't.](https://docs.google.com/forms/d/e/1FAIpQLSd-SWVgvhO7Z_jAfaV9_bFGa0dUZPuX0JORzPGS8SDP7G-dVQ/viewform)

Reviewer 2: How does this sound:
````suggestion
1. Download and extract the `elastic-profiling` CLI by replacing the `<PROVIDED_URL>` placeholder in the following example with the URL you received in your private beta onboarding call:
   ```bash
   wget -O- <PROVIDED_URL> | tar xz
   chmod +x
   ```
  If you haven't signed up for the private beta, sign up [here](https://docs.google.com/forms/d/e/1FAIpQLSd-SWVgvhO7Z_jAfaV9_bFGa0dUZPuX0JORzPGS8SDP7G-dVQ/viewform).
````
If the URL is a prereq for setting up profiling, we may want to add this to the Prerequisites section, and add this note about where to sign up there. We might not want people to get this far in the process to realize they don't have everything they need.

Author: Looks good,  @mdbirnstiehl . Thanks. 
@inge4pres - can you review and merge?
